content/1.7/back-office/shop-parameters/contact/stores/add-edit stores.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Add / Edit Stores
+weight: 1
+---
+
+# Add / Edit Store
+
+While adding or editing a store, the following fields/toggles/drop downs/parameters are displayed:
+
+1) Name
+
+This input field allows to fill the name of the store. It can be translated: When several languages are installed, a drop-down next to the field is displayed. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down. When hovered on "name", shows the following suggestion: "Store name (e.g. City Center Mall Store). Allowed characters: letters, spaces and %s"
+
+2) Address
+
+This input field allows to fill the address of the store. This field is mandatory and can be translated: When several languages are installed, a drop-down next to the field is displayed. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down.
+If the save button is pressed and if this field is empty, the following error message is displayed: "The field address1 is required at least in _default language_"
+
+3) Address (2)
+
+This input field allows to fill the second address line of the store. This field can be translated: When several languages are installed, a drop-down next to the field is displayed. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down.
+
+4) Zip/Postal code
+
+This input field allows to fill the zip/postal code of the store. The field is not marked as mandatory, but if the field is empty, and trying to save, shows the message: "A Zip/Postal code is required.". Also the postcode can't be too long, if it's too long, and trying to save shows the message: "The postcode field is too long (12 chars max).", also if the format isn't correct, when saving shows error: "The Zip/Postal code is invalid. It must be entered as follows: 00000"
```

Comment:
```suggestion
This input field allows to fill the zip/postal code of the store. The field is not marked as mandatory, but if the field is empty when trying to save, the following error message is displayed: "A Zip/Postal code is required.". If the postal code exceeds 12 characters when trying to save, the following error message is displayed: "The postcode field is too long (12 chars max).". If the format isn't correct when saving, the following error message is displayed: "The Zip/Postal code is invalid. It must be entered as follows: 00000". The format of the postal code is defined for each country in International > Locations > Countries > Add/edit country.
```

Reply:
I added missing information here about the format of the postal code: it's defined for each country in International > Locations > Countries > Add/edit country.